Enterprise-CMCS/macpro-mdct-mfp
# [MFP] SAR - R,E,&T - participants admitted to a facility from the community (9th page, period 2)

### Description



### Related ticket(s)


CMDCT-28

---
### How to test

Because this PR is dependent on reporting period, it's best to pull this locally and test vs. using the deploy link

1. Login and create a WP 
2. Create a SAR, and head to the RET section in the sidebar
3. You should only be able to see the 9th page (participants admitted to a facility from the community) within the sidebar if the SAR includes data from reporting period 2
4. Confirm that the expected headings + fields correspond to those in Figma (https://www.figma.com/file/oOJMUZerbafZefLQQ33GwI/MFP_Playground?type=design&node-id=0-1&mode=design&t=QxVlHTIyD4wbvAJ1-0)
5. Go to lines 287-288 in `formTemplates.ts` and change the condition from ```if (route.conditionallyRender === "showOnlyInPeriod2" && workPlanMetaData?.reportPeriod != 2)``` to ```if (route.conditionallyRender === "showOnlyInPeriod2" && workPlanMetaData?.reportPeriod != 1)```. This flips the logic to imply that this page will only show up when the WP was created in reporting period 1. 
6. Rerun locally, and repeat steps 1 and 2. You should no longer be able to see the 9th page in the sidebar 
7. Discard local changes! 

### Important updates




---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
